src/Http/Routing/WebRouting.php

```diff
@@ -19,12 +20,25 @@ public function wire(): void
     {
         $this->router->get("/", fn() => view("welcome"))->name("home");
 
-        $this->router->get("/auth/register", [RegisterController::class, "create"])->name("register.form");
-        $this->router->post("/auth/register", [RegisterController::class, "store"])->name("register");
-        $this->router->get("/auth/login", [LoginController::class, "store"])->name("login.form");
-        $this->router->post("/auth/login", [LoginController::class, "login"])->name("login");
+        $this->router->get("/auth/register", [RegisterController::class, "create"])->name("register");
+        $this->router->post("/auth/register", [RegisterController::class, "store"])->name("register.store");
+        $this->router->get("/auth/login", [LoginController::class, "store"])->name("login");
+        $this->router->post("/auth/login", [LoginController::class, "login"])->name("login.store");
         $this->router->get("/auth/logout", [LoginController::class, "logout"])->name("logout")->middleware("auth");
 
+        $this->router->controller(PasswordResetController::class)->group(function (): void{
+            $this->router->get("/email/verify", "create")->middleware("auth")->name("verification.notice");
+            $this->router->get("/email/verify/{id}/{hash}", "store")->middleware(["auth", "signed"])->name("verification.verify");
+            $this->router->post("/email/verification-notification", "notification")->middleware(["auth", "throttle:6,1"])->name("verification.send");
```

Comment:
Why `6,1`?

Reply:
@krzysztofrewak It means 6 requests allowed every 1 minute

Reply:
I know what's that. I asked why exactly 6 and 1?